glow_pcb_8266.yaml

```
@@ -0,0 +1,258 @@
+---
+# Home Assistant Glow
+#
+# Read your electricity meter by means of the pulse LED on your 
+# meter, useful if you do not have a serial port (P1).
+# © Klaas Schoute
+#
+substitutions:
+  device_name: glow
+  friendly_name: Glow
+  project_version: "3.2.0"
+  device_description: "Measure your energy consumption with the pulse LED on your smart meter"
+
+  # Define the GPIO pins
+  pulse_pin: GPIO5
+  status_led: GPIO2
+
+dashboard_import:
+  package_import_url: github://klaasnicolaas/home-assistant-glow/glow_pcb_8266.yaml
+
+esphome:
+  name: '${device_name}'
+  comment: '${device_description}'
+  project:
+    name: "klaasnicolaas.home-assistant-glow"
```

```suggestion
    name: "klaasnicolaas.hardware-glow-v1"
```
Or something similar?